pkg/vaildations/cluster_version.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	clusterVersionID  = "Opensift Cluster Version is valid"
+	clusterVersionID  = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/version/ocp/"
```

Comment:
You could put the `https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment` into a const in common will make the line shorter,

Reply:
This is what im suggesting :
```go 
// in common.go
const TGMEnvBaseURI = "https://github.com/redhat-partner-solutions/vse-sync-test/tree/main/tests/sync/G.8272/environment/"

// in cluster_version.go
const clusterVersionID = TGMEnvBaseURI + "version/ocp/"
```

If we have another base path add another constant. 

Depends what you mean by readable.

It is harder to look at `cluster_version.go` and see exactly what `clusterVersionID` is going to be. 

However from a I'm working in the code perspective I think this is loads more readable as it removes stuff that isn't really relevant and replaces it with a summary that actually means something.

Reply:
Done